src/odemis/util/test/transform_test.py

```diff
@@ -649,5 +651,32 @@ def test_affine_transform_inverse_known_values(self):
             numpy.testing.assert_array_almost_equal(src, tform(dst))
 
 
+class AnamorphosisTransformKnownValues(unittest.TestCase):
+
+    @staticmethod
+    def _grid(n):
```

Comment:
Please explain what kind of grid it generates. The shape is (n x n), but the content is not absolutely obvious.

Reply:
It looks like this function is equal to `odemis.util.spot.GridPoints()`

Reply:
This function is equal to `odemis.util.spot.GridPoints()`, please use that function.